ui/pages/home/home.component.js

```diff
@@ -118,6 +121,9 @@ function shouldCloseNotificationPopup({
   return shouldClose;
 }
 
+const METAMETRICS_SETTINGS_LINK =
+  'https://support.metamask.io/privacy-and-security/how-to-manage-your-metametrics-settings/';
```

Comment:
Would it be doable to move this alongside other site-specific and external URLS in `builds.yaml`?

Reply:
Fixed